images/virtualization-artifact/pkg/controller/vd/internal/source/registry.go

```diff
@@ -161,7 +176,12 @@ func (ds RegistryDataSource) Sync(ctx context.Context, vd *virtv2.VirtualDisk) (
 
 		return reconcile.Result{Requeue: true}, nil
 	case !podutil.IsPodComplete(pod):
-		log.Info("Provisioning to DVCR is in progress", "podPhase", pod.Status.Phase)
```

Comment:
Please, keep logs like this one. They are wanted.

Reply:
Sending events also writes to the log, surely it is necessary to duplicate logs?

Reply:
Ah, then it's ok :)